package.json

```diff
@@ -68,6 +69,7 @@
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1",
     "faker": "^4.1.0",
+    "gitdocs": "^2.0.0",
```

Comment:
Initially I think that it's not necessary to include gitdocs as a dependency even though the current `@next` version is broken. The `serve-docs` script won me over.